pkg/controllers/scheduling/scheduling_controller.go

```diff
@@ -36,6 +36,7 @@ import (
 const (
 	clusterSetLabel                = "cluster.open-cluster-management.io/clusterset"
 	placementLabel                 = "cluster.open-cluster-management.io/placement"
+	placementDisableAnnotation     = "cluster.open-cluster-management.io/placement-scheduling-disable"
```

Comment:
PlacementDisableAnnotationKey = "cluster.open-cluster-management.io/experimental-scheduling-disable"

Add some description here, You might want to put it in the API repo so it is easier to import.

Reply:
Done